lib/readline.js

```diff
@@ -12,7 +12,6 @@ const util = require('util');
 const inherits = util.inherits;
 const EventEmitter = require('events').EventEmitter;
 
-
```

Comment:
So far as I know, I didn't touch those lines or any code that uses those values. If they were unused, I would have thought the linter would have caught it. Maybe it doesn't lint on `const` values?